go.mod

```diff
@@ -54,6 +54,7 @@ require (
 	github.com/tendermint/go-amino v0.16.0
 	github.com/tendermint/tendermint v0.37.0-rc1
 	github.com/tendermint/tm-db v0.6.7
+	github.com/tidwall/btree v1.5.2
```

Comment:
I think we use github.com/google/btree.
What's the difference with this implementation?

Reply:
It has the `Iter()` api which give us a step iterator, which is necessary for implementing the `dbm.Iterator` interface.

Reply:
I guess missing this API is the main reason the `tm-db` has to use a go-routine to implement the iterator.